tests/test_views.py

```diff
@@ -175,6 +175,50 @@ class View(tables.SingleTableView):
         self.assertIsInstance(paginator, tables.LazyPaginator)
         self.assertEqual(paginator.orphans, 10)
 
+    def test_get_paginate_by_top_of_paginate_by(self):
+        # defined in paginator_class
```

Comment:
It seems this comment is not applicable to this test, is it possible it was accidentally copied?

Reply:
Well, It has been a long time since I made this PR. I will take a look and (probably) make some changes.

Reply:
I wasn't (and still am not!) that fluent in English. I tried to fix some problems, make them more clear, and show some use cases in tests. Hope to be useful.

P.S. Sorry about the forced pushes 😄.